### PR TITLE
Fix linting errors

### PR DESCRIPTION
**Pull request by joshnesbitt**
This PR also removes the autoprefixer from being required, as it's unused.

### CODE REVIEW

File: src/application.js [hunk 1]
```diff
@@ -21,8 +21,7 @@ export function run() {
   commander.command('init <name>').
     description('Create a new component project from called [name].').
     action((name, options) => {
-      options.name = name;
-      executeCommand(options);
+      executeCommand(Object.assign({ name }, options));
```
Comment (romidane) [inline src/application.js]: Just a functional thing `` Object.assign({ }, { name }, options)`` to return a new object ;)

**Comment by romidane** [comment]
Looks good 👍 :shipit:

File: src/application.js (hunk 1 above)
Comment (joshnesbitt) [inline src/application.js]: Good shout, but won't `{ name }` create a new object?

Comment (romidane) [inline src/application.js]: Yh it will I supposed :). Good point 😄

Comment (joshnesbitt) [inline src/application.js]: 😄

**Comment by joshnesbitt** [comment]
Thanks for the review @romidane.

**Pull request merged**